packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -1,16 +1,53 @@
 import { DesignSystem } from '@microsoft/fast-foundation';
+import { attr } from '@microsoft/fast-element';
 import { styles } from '../base/styles';
 import { template } from '../base/template';
 import type { TableNumberField } from '../../table/types';
 import { TableColumnTextBase } from '../text-base';
-import { TableColumnSortOperation } from '../base/types';
+import { TableColumnSortOperation, TableColumnValidity } from '../base/types';
 import { tableColumnDateTextGroupHeaderTag } from './group-header-view';
 import { tableColumnDateTextCellViewTag } from './cell-view';
 import type { ColumnInternalsOptions } from '../base/models/column-internals';
+import type {
+    DateTextFormat,
+    LocaleMatcherAlgorithm,
+    EraFormat,
+    SimpleNumberFormat,
+    TimeZoneFormat,
+    FormatMatcherAlgorithm,
+    DayPeriodFormat,
+    DateStyle,
+    TimeStyle,
+    HourCycle,
+    MonthFormat,
+    WeekdayFormat
+} from './types';
+import { TableColumnDateTextValidator } from './models/table-column-date-text-validator';
 
 export type TableColumnDateTextCellRecord = TableNumberField<'value'>;
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface TableColumnDateTextColumnConfig {}
+export interface TableColumnDateTextColumnConfig {
```

Comment:
Do all of these properties need to be part of the column config? I would have expected just the formatter to be there, and then the other properties influence how the settings with which the formatter is created.

Reply:
That's a good point. We don't need them in there anymore.